tests/test_common.py

```diff
@@ -770,3 +770,23 @@ def test_private_notes_from_595__9():
     )  # record/1005469
 
     assert '_private_notes' not in hepnames.do(create_record(snippet))
+
+
+def test_legacy_version_from_005():
+    schema = load_schema('hep')
+    subschema = schema['properties']['legacy_version']
+
+    snippet = (
+        '<controlfield tag="005">20180919130452.0</controlfield>'
+    )
```

Comment:
can you add a comment with the recid you took it from (not super useful here, but to follow the convention of the tests)?